### PR TITLE
Measuring Accuracy

### PR DESCRIPTION
**Pull request by sagedemage**
Modified the training model program to measure the accuracy of the model during the evaluation of the model.

**Extra Tasks Done**
1. Fixed the `train_one_epoch` function where it should be iterating to all of the batches of the training loader. When this function finishes, it should return the correct average loss. (bug)
2. Tweaked SGD Optimization
    1. Tweaked the epochs, and learning rate.
    2. Set the momentum of the optimizer to 0.9.
3. Modified the script to measure the accuracy of the movie recommendation program 1000 times.
4. Added links to resources that were useful for this project.

### CODE REVIEW

**Review by sagedemage — COMMENTED** [review]
LGTM!
